src/cdk-experimental/menu/menu.ts

```diff
@@ -34,6 +36,7 @@ export class CdkMenu {
   /** Event emitted when the menu is closed. */
   @Output() readonly closed: EventEmitter<void | 'click' | 'tab' | 'escape'> = new EventEmitter();
 
-  /** Emits the activated element when checkbox or radiobutton state changed  */
-  @Output() change: EventEmitter<CdkMenuItem>;
+  /** All the child MenuItem components which this directive wraps including descendants */
+  @ContentChildren(CdkMenuItem, {descendants: true})
+  readonly _allItems: QueryList<CdkMenuItem>;
```

Comment:
Typically, we'd mark the field as '_allItems!: QueryList<...>', and assert the field is not undefined in ngAfterContentInit(). But I don't know if that's the convention in the CDK.

Reply:
Yeah, we don't use that convention yet because we haven't enabled `--strictPropertyInitialization` w/ TypeScript. We will likely do this someday, but don't have the assertion system set up presently.